vignettes/themeBuilder_addin.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Theme Configuration Builder"
+author: "Mohammed Ali"
+date: "`r Sys.Date()`"
+output: 
+    rmarkdown::html_vignette:
+        toc: TRUE
+        toc_depth: 3
+vignette: >
+  %\VignetteIndexEntry{Theme Configuration Builder}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Overview
+
+To help package users in creating valid theme configuration file
```

Comment:
The period "." is missing at the end of this sentence, and similarly for many sentences throughout the vignette.

Reply:
done, I will do more deep review for all articles before submitting it to CRAN